jbrowse/src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantWidget/ExtendedVariantWidget.js

```diff
@@ -31,71 +33,323 @@ export default jbrowse => {
+                                </div>
+                            </div>
+                        )
+                    }else if(feat["INFO"][displays[display].properties[property]].length > 1){
+                        var children = []
+                        for(var val in feat["INFO"][displays[display].properties[property]]){
+                            children.push(
+                                <div className={classes.fieldSubValue}>
+                                    {feat["INFO"][displays[display].properties[property]][val]}
+                                </div>
+                            )
+                        }
+                        tempProp.push(
+                            <div className={classes.field}>
+                                <div className={classes.fieldName}>
+                                    {displays[display].properties[property]}
+                                </div>
+                                {children}
+                            </div>
+                        )
+                    }
                 }
             }
-            propertyJSX.push(tempProp)
+            if(tempProp.length != 0){
+                propertyJSX.push(tempProp)
+            }
         }
-
         var displayJSX = []
-        for(var i = 0; i < displays.length; i++){
+        for(var i = 0; i < propertyJSX.length; i++){
             displayJSX.push(
                 <BaseCard title={displays[i].name}>
-                   <div style={{ padding: '7px', width: '100%', maxHeight: 600, overflow: 'auto' }}>
-                        <Table className={displays[i].name}>
-                            <TableHead>
-                                {propertyJSX[i]}
-                            </TableHead>
-                        </Table>
-                   </div>
+                    {propertyJSX[i]}
                 </BaseCard>
             )
         }
         return displayJSX
     }
 
+    function makeChart(samples, ref, alt, classes){
+        const [state, setState] = useState(null)
+        useEffect(() => {
+            setState(
+                <BaseCard title="Genotypes">
+                    <div>
+                    Loading genotypes...
+                    </div>
+                </BaseCard>
+            )
+            var alleleCounts = {}
+            var alleleTotal = 0
+            alleleCounts[ref] = 0
+            for(var i in alt){
+                alleleCounts[alt[i]] = 0
+            }
+            var gtCounts = {}
+            var gtTotal = 0
+            for(var sample in samples){
+                var gt = samples[sample]["GT"]
+                for(var entry in gt){
+                    // CASES
+                    // ./. -- no call
+                    if(gt[entry] == "./."){
+                        if(gtCounts["No Call"]){                          // if gtCounts entry is not null, or we have a preexisting entry for it
```

Comment:
should we have a constant for the string 'No Call'?

Reply:
Updated